tailwind-plugins/form-control.js

```diff
@@ -18,12 +18,12 @@ function formControl(theme) {
   return {
     display: "block",
     width: "100%",
-    height: "calc(1.5em + .75rem + 2px)",
-    padding: ".375rem .75rem",
+    height: "calc( 2.5rem + 2px)", // 42px if 1rem = 16px
```

Comment:
Based on values in our design system: https://www.figma.com/file/sOxNGYMzwZT0B9Fal2D4WT/%5BWIP%5D-Component-Library?node-id=1232%3A11585&mode=dev